packages/ra-keycloak/package.json

```
@@ -19,8 +19,8 @@
     },
     "dependencies": {
         "jwt-decode": "^3.1.2",
-        "keycloak-js": "^23.0.1",
-        "react-admin": "^4.4.0"
+        "keycloak-js": "^26.0.0",
+        "react-admin": "^5.1.1"
```

You went through all the trouble of replacing `react-admin` by `ra-core`, so you earned the right to replace this dep too I believe :grin: 

```suggestion
        "ra-core": "^5.1.1"
```